OpenEdXMobile/res/layout/fragment_course_unit_video.xml

```diff
@@ -53,5 +53,12 @@
             tools:targetApi="17" />
     </LinearLayout>
 
+    <ListView
```

Comment:
I had this in mind when I was implementing this feature, the only reason that I resorted to ListView was cuz it gave the `smoothScrollToPosition` function and was easy to implement overall.

If its not too much effort, I will look into using `RecyclerView`.